src/Lykke.Service.HFT.Core/Constants.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace Lykke.Service.HFT.Core
+﻿using System;
+
+namespace Lykke.Service.HFT.Core
 {
     public static class Constants
     {
         public const string ComponentName = "HFT API (high-frequency-trading-service)";
+
+        public static ulong OrderCounter;
```

Comment:
This is not a production-like performance counters; they should not be in test / master branches.